lib/iris/coords.py

```diff
@@ -963,10 +963,10 @@ def collapsed(self, dims_to_collapse=None):
                 for index in np.ndindex(shape):
                     index_slice = (slice(None),) + tuple(index)
                     bounds.append(serialize(self.bounds[index_slice]))
-                dtype = np.dtype('S{}'.format(max(map(len, bounds))))
+                dtype = np.dtype('U{}'.format(max(map(len, bounds))))
                 bounds = np.array(bounds, dtype=dtype).reshape((1,) + shape)
             points = serialize(self.points)
-            dtype = np.dtype('S{}'.format(len(points)))
+            dtype = np.dtype('U{}'.format(len(points)))
```

Comment:
These change the Python 2 behaviour. i.e. If I collapse a cube and get the value of a textual, collapsed coordinate it will now be an instance of `numpy.unicode_` instead of an instance of `numpy.string_`. User code doing `isinstance(value, str)` will fail.